components/org.wso2.carbon.business.rules.web/src/components/BusinessRuleCreator.jsx

```diff
@@ -68,23 +65,27 @@ class BusinessRuleCreator extends React.Component {
                         <Grid item xs={12}>
                             <Grid container justify="center" spacing={Number(styles.spacing)}>
                                 <Grid item>
-                                    <CreateButton
-                                        onClick={(e) => BusinessRulesUtilityFunctions.loadTemplateGroupSelector(
-                                            BusinessRulesConstants.BUSINESS_RULE_TYPE_TEMPLATE)}
-                                        mode={BusinessRulesConstants.BUSINESS_RULE_TYPE_TEMPLATE}
-                                        title='From Template'
-                                        description='Create a business rule based on an existing template'
-                                    />
+                                    <Link
+                                        to={"/business-rules/templateGroupSelector/" + BusinessRulesConstants.BUSINESS_RULE_TYPE_TEMPLATE}
+                                        style={{textDecoration: 'none'}}>
+                                        <CreateButton
+                                            mode={BusinessRulesConstants.BUSINESS_RULE_TYPE_TEMPLATE}
+                                            title='From Template'
+                                            description='Create a business rule based on an existing template'
+                                        />
+                                    </Link>
                                 </Grid>
                                 <Grid item>
-                                    <CreateButton
-                                        onClick={(e) => BusinessRulesUtilityFunctions.loadTemplateGroupSelector(
-                                            BusinessRulesConstants.BUSINESS_RULE_TYPE_SCRATCH)}
-                                        mode={BusinessRulesConstants.BUSINESS_RULE_TYPE_SCRATCH}
-                                        title='From Scratch'
-                                        description='Create a business rules with templates for input & output,
+                                    <Link
```

Comment:
If you always going to wrap the CreateButton with Link better to move that part into the component itself and pass the URL as a prop.